x-pack/platform/plugins/private/intercepts/public/prompter/service/telemetry/event_definitions.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { type RootSchema, type EventTypeOpts } from '@elastic/ebt/client';
+
+export enum EventMetric {
+  INTERCEPT_TERMINATION_INTERACTION = 'intercept_termination_interaction',
+  INTERCEPT_REGISTRATION = 'intercept_registration',
+  INTERCEPT_OVERLOAD = 'intercept_overload',
+}
+
+export enum EventFieldType {
+  INTERACTION_TYPE = 'interaction_type',
+  INTERCEPT_ID = 'intercept_id',
```

Comment:
Not super important, but it could be helpful to have a "duration" field to measure the amount of time the user spends interacting with the prompter.

Reply:
Thanks Tim, I think this would be helpful regardless of if it's a completion or dismissal event